bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/CodeType.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.openthermgateway.internal;
+
+/**
+ * The {@link CodeType} field is not part of OpenTherm specification, but added by OpenTherm Gateway.
+ * It can be any of the following:
+ *
+ * T: Message received from the thermostat
+ * B: Message received from the boiler
+ * R: Request sent to the boiler
+ * A: Response returned to the thermostat
+ * E: Parity or stop bit error
+ *
+ * @author Arjen Korevaar - Initial contribution
+ * @author James Melville - Introduced code filtering functionality
+ */
+public enum CodeType {
+    /**
+     * Message received from the thermostat
+     */
+    T,
```

Comment:
Can you give these enum value more verbose names?

Reply:
Hi cpmeister, some of the code of the OTGW binding is directly related to the OpenTherm or the OTGW specifications. While it is possible to use more verbose names, this would require extra mappings and in general obfuscates the relationship to the OpenTherm or OTGW specifications. I would prefer to stick with the values as they are used by the OTGW. The comments should provide sufficient explanation of each value.